### ISSUE TITLE
TypedArray - ES6 note is duplicated

### ISSUE THREAD

**Issue by Phosra** [mdn/content#18762]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray

### What specific section or headline is this issue about?

Examples

### What information was incorrect, unhelpful, or incomplete?

This very well may be a non-issue, but I observed that the first example is duplicated in each subclass constructors' documentation, e.g. see [/docs/Web/JavaScript/Reference/Global_Objects/Int8Array/Int8Array#description](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Int8Array/Int8Array#description).

For clarification, it is in the MDN URL referenced above, *and* each of the constructors' pages

### What did you expect to see?

It makes sense to be duplicated across all of the constructors, or put in the global page, but unnecessarily duplicated into both?

### Do you have anything more you want to share?

I do suppose that this is related to #18002.

This was also reworded in #18707 (cc @Josh-Cena), so if this note is being kept, we may want to reword it too..?

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/javascript/reference/global_objects/typedarray`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/javascript/reference/global_objects/typedarray/index.md
* Last commit: https://github.com/mdn/content/commit/2e2eab54121a9f521d5ea9438c635a76dfdfe255
* Document last modified: 2022-07-22T09:29:58.000Z

</details>

**Comment by Josh-Cena** [comment]
I'm okay with duplicating it in both places, or removing it from the `TypedArray` page, but I don't have great ideas about how to do this before #18002 either.

**Comment by Isaac108** [comment]
I don't believe this is an issue. I believe this was done for the sake of clarification or a cordial reminder.

**Comment by Josh-Cena** [comment]
It doesn't have to be duplicated everywhere now—ES6 has been 7 years and I doubt a lot of people even know that you used to be able to construct typed arrays without `new`. Since we have a BCD entry already, it may even make sense to remove it from all places except the generic TypedArray page.
